### PR TITLE
DSLAB: added conference references

### PR DESCRIPTION
**Pull request by ineiti**
Just in case you want to see the updates...

### CODE REVIEW

File: data/DSLAB/projects.yaml [hunk 1]
```diff
@@ -189,9 +234,30 @@ projects:
...
+            notes:
+              - label: Published in
+                text: "SoCC 2010"
+          - type: Paper
+            title: "Cloud9: A Software Testing Service"
+            url: http://dslab.epfl.ch/pubs/cloud9-ladis.pdf
+            notes:
+              - label: Published in
+                text: "ACM Operating Systems Review 2009"
+                url: http://www.sigops.org/osr.html
```
Comment (tharvik) [inline data/DSLAB/projects.yaml:257]: It's a 404 for me

File: data/DSLAB/projects.yaml [hunk 2]
```diff
@@ -237,5 +311,13 @@ projects:
...
             - Verification
         type: Application
+        information:
+          - type: Paper
+            title: "Failure Sketching: A Technique for Automated Root Cause Diagnosis of In-Production Failures"
+            url: https://dslab.epfl.ch/pubs/gist.pdf
+            notes:
+              - label: Published in
+                text: "SOSP '15"
+                url: http://dl.acm.org/authorize?N93296
```
Comment (tharvik) [inline data/DSLAB/projects.yaml:321]: The others notes are using the "resolved" link
```suggestion
                url: https://dl.acm.org/doi/10.1145/2815400.2815412
```

File: data/DSLAB/projects.yaml (hunk 1 above)
Comment (cgrigis) [reply data/DSLAB/projects.yaml:257]: Same here.
It seems available at: https://dl.acm.org/toc/sigops/2010/43/4

File: data/DSLAB/projects.yaml [hunk 3]
```diff
@@ -189,9 +234,30 @@ projects:
...
+          - type: Paper
+            title: "Parallel Symbolic Execution for Automated Real-World Software Testing"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "EuroSys 2011"
+                url: http://eurosys2011.cs.uni-salzburg.at/pdf/eurosys2011-salomie.pdf
+          - type: Paper
+            title: "Automated Software Testing as a Service"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
```
Comment (cgrigis) [inline data/DSLAB/projects.yaml:247]: I believe this is the wrong link (a copy/paste from the previous one — yes, it is wrong on the Cloud9 page as well).
The correct link is: https://dslab.epfl.ch/pubs/taas.pdf

File: data/DSLAB/projects.yaml [hunk 4]
```diff
@@ -189,9 +234,30 @@ projects:
...
+            notes:
+              - label: Published in
+                text: "SoCC 2010"
```
Comment (cgrigis) [inline data/DSLAB/projects.yaml:250]: ... and the URL for the conf is: https://dl.acm.org/doi/proceedings/10.1145/1807128
(it is in the "Benchmarking & testing" session)

**Pull request merged**